src/compat/event_listeners.ts

```diff
@@ -101,7 +102,7 @@ function eventPrefixed(eventNames : string[], prefixes? : string[]) : string[] {
   return eventNames.reduce((parent : string[], name : string) =>
     parent.concat((prefixes == null ? BROWSER_PREFIXES :
                                       prefixes)
-          .map((p) => p + name)), []);
+      .map((p) => p + name)), []);
```

Comment:
Got the feeling that it is less readable and that it should be possible to align multiple function calls.

Reply:
Yes, that's the big problem with the current indentation rules.

I'm not sure if we can make `eslint` understand that.